### PR TITLE
Commit Lint CI & git hook

### PR DESCRIPTION
**Pull request by diced**

### CODE REVIEW

**Review by Vetlix — COMMENTED** [review]
good job, only like 30 commits

**Comment by diced** [comment]
finally after github being dumb

**Review by TacticalTechJay — CHANGES_REQUESTED** [review]
I don't get it. :D

**Review by advilm — APPROVED** [review]
Bad execution, but the result is good

File: .github/workflows/ci-commits.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: 'CI: Commit Lint'
+on: [push, pull_request]
+jobs:
+  commit-lint:
+    name: Commit Lint
+    # if: "!(contains(github.event.head_commit.message, '[skip_ci]'))"
```
Comment (tbnritzdoge) [inline .github/workflows/ci-commits.yml]:
```suggestion
```

**Review by tbnritzdoge — CHANGES_REQUESTED** [review]

**Review by TacticalTechJay — APPROVED** [review]

**Review by kaihodev — APPROVED** [review]
Let us see how this goes（ゝ。∂)～☆

**Review by TacticalTechJay — APPROVED** [review]
The moment of truth.

**Review by Vetlix — APPROVED** [review]

**Review by advilm — APPROVED** [review]
:+1:

**Pull request merged**